docs/Serverless.md

```diff
@@ -0,0 +1,25 @@
+# Serverless
```

Comment:
@namanshah76 - this is a good start. We need to cover the pricing concepts - SCUs, how we suspend/resume, parked DBs, how auto-everything works, Add a note that it is Tech preview, etc.